lib/badge/badge.tsx

```diff
@@ -2,13 +2,52 @@ import React from 'react';
 import BEM, { BEMInterface } from '../utilities/ts/bem';
 import getLabel from './utilities/get-label';
 
+type ModifierTypes = `error` | `success`;
+
 interface Props {
   className?: string;
-  modifiers?: string;
+  modifiers?: ModifierTypes;
   style?: React.CSSProperties;
-  value?: string | number;
+  value?: number | string;
 }
 
+/**
+ *
+ * @param props
+ *
+ * The following `props` are available to the `<Badge />` component.
+ *
+ * | Name               | Type(s)              | Arguments              |
+ * | :----------------- | :------------------- | :--------------------- |
+ * | `props.className?` | `string`             |                        |
+ * | `props.modifiers?`  | `string`             | `"error"`, `"success"` |
+ * | `props.style?`     | `CSS properties`     |                        |
+ * | `props.value?`     | `number` \| `string` |                        |
+ *
+ * @returns
+ *
+ * ```
+ * <!-- Declaration -->
+ * <Badge />
+ *
+ * <!-- Output -->
+ * <span class="uds-badge" unselectable="on">0</span>
+ *
+ * <!-- Declaration -->
+ * <Badge modifiers="success" value={100} className="pad-1-rem" />
+ *
+ * <!-- Output -->
+ * <span class="uds-badge uds-badge--success pad-1-rem" unselectable="on">99+</span>
+ *
+ *  <!-- Declaration -->
+ * <Badge modifiers="error" value={12} style={{ zIndex: '9999' }}/>
+ *
+ * <!-- Output -->
+ * <span class="uds-badge uds-badge--error" style="z-index:9999" unselectable="on">9+</span>
```

Comment:
HI @DanMad , Same as above. the value you are passing is 12. But the output is showing 9+

Reply:
Same explanation as above ;)